examples/egg_parameters.yml

```diff
@@ -0,0 +1,79 @@
+name: EGG
+
+flownet:
+  data_source:
+    input_case: ../input_model/EGG_MODEL_ECL
+  cell_length: 10
+  additional_flow_nodes: 50
+  additional_node_candidates: 1000
+  hull_factor: 1.2
+  random_seed: 999
+  perforation_handling_strategy: bottom_point #bottom_point #multiple
+  fast_pyscal: True
+  training_set_fraction: 0.8 # Fraction of observations (from 0 to 1) to use as training data
+  fault_tolerance: 0.1
+ert:
+  static_include_files: ../input_model/
+  realizations:
+    num_realizations: 250
+    required_success_percent: 80
+    max_runtime: 600
+  queue:
+    system: LOCAL
+    name: mr
```

Comment:
For the local queue this doesn't have to be specified.
```suggestion
```

Reply:
I see, so we specify "name" if we are running in a server?

Reply:
`mr` here is an LSF queue name to be used in ERT, we don't use that here.